genai-perf/genai_perf/metrics/telemetry_metrics.py

```diff
@@ -39,12 +39,12 @@ class TelemetryMetrics:
     """
 
     TELEMETRY_METRICS = [
-        MetricMetadata("gpu_power_usage", "watts"),
-        MetricMetadata("gpu_power_limit", "watts"),
-        MetricMetadata("energy_consumption", "joules"),
-        MetricMetadata("gpu_utilization", "percentage"),
-        MetricMetadata("total_gpu_memory", "bytes"),
-        MetricMetadata("gpu_memory_used", "bytes"),
+        MetricMetadata("gpu_power_usage", "W"),
+        MetricMetadata("gpu_power_limit", "W"),
+        MetricMetadata("energy_consumption", "MJ"),
```

Comment:
Is this joules or megajoules? Looks like the units changed.

Reply:
yeah, it's MJ. The energy_consumption, and memory related metrics will be scaled for readability.
The scaling part will be in TelemetryStatistics class just like how scaling is done right not in statistics class.